<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaMain</class>
 <widget class="QMainWindow" name="ventanaMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">#frame{background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0.440678 rgba(85, 0, 127, 255), stop:1 rgba(255, 255, 255, 255));}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius:18px;
}

QPushButton:hover{
	background-color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>600</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="boton_minimizar">
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imgs/minimizar.png</normaloff>imgs/minimizar.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="boton_cerrar">
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imgs/eliminar.png</normaloff>imgs/eliminar.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget_3">
         <property name="styleSheet">
          <string notr="true">#frame{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(85, 0, 127, 255), stop:1 rgba(255, 255, 255, 255));}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="ventana_inicio">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>149</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>600</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
	color: rgb(255,255,254);
	font: 12pt&quot;Arial&quot;;
	background-color: rgba(0,0,0,0);
}

QLineEdit{
background-color: rgb(0,0,0,0);
border-radius:10px;
font: 12pt&quot;Arial&quot;;
border: 2px solid rgb(120,0,171);
}

QPushButton{
border-radius:10px;
font: 12pt&quot;Arial&quot;;
color: rgb(255,255,254);
border:2px solid #ffffff;
}

QPushButton:hover{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color: rgb(255,255,254);
border: 2px solid rgb(120,0,171);
	background-color: rgb(120,0,171);
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 16pt&quot;Arial&quot;;</string>
                </property>
                <property name="text">
                 <string>Iniciar Aplicaci√≥n</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="usuarioE">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="pw">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="ingresarUsuario">
                <property name="text">
                 <string>Ingresar usuario</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QSplitter" name="splitter">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Agregar usuario:   </string>
                 </property>
                </widget>
                <widget class="QPushButton" name="agregarUsuario">
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
border-radius: 10px;
font: 12pt&quot;Arial&quot;;
color: rgb(255,255,254);
border:None;
background-color: None;
}

QPushButton:hover{
border-radius:10px;
font: 12pt&quot;Arial&quot;;
	color: rgb(117,0,175);
border: None;
background-color: None;
}</string>
                 </property>
                 <property name="text">
                  <string>Agregar</string>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>148</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="ver_tabla">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(85, 0, 127, 255), stop:1 rgba(255, 255, 255, 255));</string>
          </property>
          <widget class="QTableWidget" name="tableView">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>110</y>
             <width>511</width>
             <height>341</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="id">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>80</y>
             <width>81</width>
             <height>20</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font:&quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="nombre">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>80</y>
             <width>113</width>
             <height>20</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font:&quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="edad">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>80</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font:&quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="apellido">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>80</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font:&quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
          </widget>
          <widget class="QPushButton" name="ingreso">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>80</y>
             <width>181</width>
             <height>23</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font:&quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Guardar informaci√≥n paciente</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="id_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>241</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font:&quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buscar_usuario">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>20</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt&quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>imgs/buscar.png</normaloff>imgs/buscar.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="buscar_usuario_v">
          <widget class="QTableWidget" name="tableView_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>521</width>
             <height>401</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="buscar">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>301</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font:&quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buscar_usuario_2">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>20</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt&quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="archivoQrc.qrc">
             <normaloff>:/fondo/imgs/buscar.png</normaloff>:/fondo/imgs/buscar.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="borrar_usuario_v">
          <widget class="QLineEdit" name="id_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>301</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font:&quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
          </widget>
          <widget class="QTableWidget" name="tableView_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>521</width>
             <height>401</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
          </widget>
          <widget class="QPushButton" name="borrar_usuario_2">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>20</y>
             <width>51</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt&quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>imgs/delete.png</normaloff>imgs/delete.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="archivoQrc.qrc"/>
 </resources>
 <connections/>
</ui>
